assets/Gruntfile.js

```diff
@@ -1,5 +1,6 @@
 // Generated on 2014-09-12 using generator-angular 0.9.8
 'use strict';
+/* jshint unused: false, camelcase: false */
```

Comment:
can we also remove file-specific things we set globally like `camelcase` and `unused`?